Dread: Add warning about custom text in FAQ (#5352)
randovania/randovania: CHANGELOG.md

### Metroid Dread

- Added: Enky and Charge Beam Doors can be made immune to Power Bombs. This is enabled in the Starter Preset, and can be toggled in Preset -> Game Modifications -> Other -> Miscellaneous -> Power Bomb Limitations.

> - Added: Warning in the FAQ section about custom text only fully displayed when running the game in English.

Reviewer: ```suggestion
- Added: Warning in the FAQ about custom text not displaying if the game is played in languages other than English.
```
